WordPress/Classes/Utility/CoreDataHelper.swift

```diff
@@ -201,6 +201,19 @@ extension CoreDataStack {
         }
     }
 
+    /// Perform a query using the `mainContext` and return the result.
+    ///
+    /// This is a re-implemenation of https://developer.apple.com/documentation/coredata/nsmanagedobjectcontext/3802019-performandwait,
+    /// which is available from iOS 15.0 (hence the deprecation attribute below).
+    @available(iOS, deprecated: 15.0, message: "Use `mainContext.performAndWait` instead.")
+    func performQuery<T>(_ block: @escaping (NSManagedObjectContext) -> T) -> T {
+        var value: T! = nil
+        self.mainContext.performAndWait {
```

Comment:
Nitpick: `self` should not be necessary, right?

```suggestion
        mainContext.performAndWait {
```

Reply:
If we want to enforce "no unneeded `self` references", we should add that rule to SwiftLint!

Reply:
Great point!